crates/daphne-worker/Cargo.toml

```diff
@@ -16,13 +16,21 @@ description = "Workers backend for Daphne"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
+axum.workspace = true
+axum-extra = { workspace = true, features = ["typed-header"] }
+axum-macros = "0.4.1"
```

Comment:
Move this to the the `Cargo.toml` for the workspace? Here and below.

Reply:
this can actually be deleted, it's a tool to help debug stuff